devito/types.py

```diff
@@ -606,7 +606,9 @@ def indexify(self, indices=None):
         if indices is not None:
             return Indexed(self.indexed, *indices)
 
-        subs = dict([(i.spacing, 1) for i in self.indices])
+        # Only replace spacing -> 1 if used as index
```

Comment:
why this change?

Reply:
I "fixed" `SparseFunction` tthat was forcing `indexify` in inject/interpolate to make this function happy.